js/login.js

```diff
@@ -0,0 +1,105 @@
+const loginEmail = document.getElementById("login-email");
+const emailAlert = document.getElementById("login-email-alert");
+const loginPassword = document.getElementById("login-password");
+const passwordAlert = document.getElementById("login-password-alert");
+const loginBtn = document.getElementById("login-submit");
+const toggleBtn = document.getElementById("toggle-password");
+const eyeIcon = document.getElementById("eye-icon");
+
+// 로그인 버튼 상태 업데이트 함수
+const loginBtnUpdate = () => {
+  // loginBtn이 disabled 되는 조건
+  const isEmailEmpty = loginEmail.value.trim() === "";
+  const isPasswordEmpty = loginPassword.value.trim() === "";
+  const isEmailError = !emailAlert.classList.contains("hidden");
+  const isPasswordError = !passwordAlert.classList.contains("hidden");
+
+  // 조건 중 하나라도 만족하면 버튼 비활성화
+  if (isEmailEmpty || isPasswordEmpty || isEmailError || isPasswordError) {
+    loginBtn.disabled = true;
+  } else {
+    loginBtn.disabled = false;
+  }
+};
```

Comment:
💊 제안
주석이 필요하다고 생각되는 경우 이를 함수로 분리하시면 주석없이도 더 명확하게 코드를 작성할 수 있습니다.

```suggestion
const isFormValidation = () => {
  const isEmailEmpty = loginEmail.value.trim() === "";
  const isPasswordEmpty = loginPassword.value.trim() === "";
  const isEmailError = !emailAlert.classList.contains("hidden");
  const isPasswordError = !passwordAlert.classList.contains("hidden");
  
  return isEmailEmpty && isPasswordEmpty && isEmailError && isPasswordError;
}

const loginBtnUpdate = () => {
  loginBtn.disabled = !isFormValidation();
};
```